### PR TITLE
adapt some __repr__ methods

### PR DESCRIPTION
**Pull request by spaceone**
just a very small PR.

### CODE REVIEW

File: circuits/core/events.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def __repr__(self):
 
         data = "%s %s" % (
             ", ".join(repr(arg) for arg in self.args),
-            ", ".join("%s=%s" % (k, repr(v)) for k, v in self.kwargs.items())
+            ", ".join("%s=%r" % (k, v) for k, v in self.kwargs.items())
```
Comment (prologic) [inline circuits/core/events.py]: Why not use ``repr()``? Does Python 3 even have ``%r``?

Comment (spaceone) [inline circuits/core/events.py]: of course python3 has  %r. Why doing explicit call of repr() if string interpolation is already able to do it?

Comment (prologic) [inline circuits/core/events.py]:
```#!python
>>> print("{0:r}".format("foo"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: Unknown format code 'r' for object of type 'str'
>>> print("%r" % "foo")
'foo'
```

This is why I don't use it I guess; I'm in favour of ``str.format()`` over ``'...' % ...``

Comment (spaceone) [inline circuits/core/events.py]: str.format makes code very ugly btw imho

Comment (prologic) [inline circuits/core/events.py]: Zen of Python; Rule #2:

"Explicit is better than implicit."

:)

Comment (prologic) [inline circuits/core/events.py]: @spaceone I guess from a readability perspective I find ``str.format()`` easier on the eyes when reading code as well as ``{0}`` easier to read vs. ``%s``. I have actually developed a habit of converting any ``"foo %s" % "bar"`` to ``"foo {0}".format("bar")`` so if I merge this in; it'll likely get changed! Happy to incorporate the other fixes!

**Comment by spaceone** [comment]
As you wished I did not change the str → repr() "explicity". (I sill find this ugly and unnecessary).
The rest has been committed to master.

**Comment by prologic** [comment]
I guess it doesn't matter too much. Too be honest I'm finding ``{foo}`` more readable these days; i.e: explicitly naming string formatting arguments.

**Pull request closed without merging**